clix/clix.py

```diff
@@ -33,14 +44,16 @@ def OnKeyPress(event):
         active = 0
         prev_Key = None
 
-    elif event.Key == 'c' and prev_Key == 'Control_L':
-        text = xerox.paste(xsel=True)
+    elif event.Key == 'c' or event.Key == 'C' and prev_Key == key_binding[0]:
```

Comment:
You are allowing the user to copy text using different combination than the default. Suppose the user has defined `RALT+SPACE` as the binding to open the clipboard, then in your code, the user will be able to copy text using `RATL+c/C` and it will ignore `LCTRL+c`. Is this behavior desired?